Tools/CISupport/ews-build-webserver/master.cfg

```diff
@@ -0,0 +1,125 @@
+import importlib
+import json
+import os
+import socket
+import sys
+
+from buildbot.plugins import reporters, util
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+loadConfig = importlib.import_module("ews-build.loadConfig")
```

Comment:
Much of this is copied from `ews-build`, right? Can we consider trying to share more of this code? I know in our other multi-master setups, duplication of code in `master.cfg` has been a source of problems.

Reply:
Can probably look into de-duplication later-on. In our other multi-master setup, we never tried de-duplication in this file earlier. Also, I don't think duplication was a source of problem, the problem was specific things (like missing out required schedulers on one master).